server.js

```diff
@@ -1,23 +1,82 @@
 const express = require("express");
 const cors = require("cors");
+const uuid = require("uuid");
 
 const app = express();
+const port = process.env.PORT || 4500;
+
+// creating increment unique id for each message
+let generateNewID = 0;
 
 app.use(cors());
+app.use(express.json());
+app.use(express.urlencoded({ extended: false }));
+
+const welcomeMessage = [];
 
-const welcomeMessage = {
-  id: 0,
-  from: "Bart",
-  text: "Welcome to CYF chat system!",
-};
+// Time formating
+const today = new Date();
+const format = Intl.DateTimeFormat("en-us", {
+  dateStyle: "full",
+  timeStyle: "medium",
+  //dayPeriod: "long"
+});
 
+console.log(format);
 //This array is our "data store".
 //We will start with one message in the array.
 //Note: messages will be lost when Glitch restarts our server.
-const messages = [welcomeMessage];
+
+const messages = welcomeMessage;
+
+// app.get("", function (request, response) {
+//   console.log(GenerateUniqueId)
+//   response.send({ welcomeMessage });
+// });
 
 app.get("/", function (request, response) {
   response.sendFile(__dirname + "/index.html");
 });
 
-app.listen(process.env.PORT);
+app.post("/allMessages", (req, res) => {
+  const message = req.body;
+  generateNewID++;
+
+  const newMessages = {
+    id: generateNewID,
+    from: message.from,
+    text: message.text,
+    time: new Date().toUTCString(),
+    //
+  };
+
+  const valid = !!message.text && !!message.from;
```

Comment:
Why the double negitives? SHould be enought just to do message.text && message.from